backend/src/api/osrm.py

```diff
@@ -0,0 +1,35 @@
+import requests
+from typing import Tuple
+from enum import Enum
+
+
+class TravelOptions(Enum):
+    WALK = "foot"
+    CAR = "car"
+    BIKE = "bike"
+
+
+osrm_endpoint = "http://router.project-osrm.org/route/v1/"
+travel_time_query = "{0}/{1},{2};{3},{4}?overview=false"
+
+
+def determine_travel_time(start: Tuple[float, float], end: Tuple[float, float], type: TravelOptions) -> float:
+    response = requests.get(osrm_endpoint + travel_time_query.format(
+        type.value, start[1], start[0], end[1], end[0]
+    ))
+
+    if response.status_code != 200:
```

Comment:
Do we want to just accept 200 only? Does the spec specify that a successful response is always 200, or should we be checking for a successful response 2XX?